client/src/helpers/APICalls/deleteBoard.ts

```diff
@@ -0,0 +1,17 @@
+import { IFetchOptions, IBoard } from '../../interface';
+
+const deleteBoard = async (board: IBoard): Promise<IBoard> => {
```

Comment:
You could delete by passing in just the `id` of the board instead of the whole board itself, which is used in `
return await fetch('boards/${board._id}', fetchOptions)`